GaugeChart.vue

```diff
@@ -105,6 +98,18 @@ export default {
         }
       }
     });
+    this.setChartData(this.last);
+  },
+  methods: {
+    setChartData(latest) {
```

Comment:
The return value from this function is always falsey, it returns `false` or `undefined`. Is this your intention?

Reply:
hmmm not sure on this one actually. this eventually bounces up to a prop in canvas which I use so we can use a computed property over a watched property (the Vue docs recommend computed over watched and this ensures the chart data updates in the computed cycle with the percentage and not later in the watch cycle). I will actually add a return true to the end of this so a successful update returns true although we are not really using that anywhere. I guess I need to look up how Vue handles props that are false or undefined, but I suppose it should return true on a successful update.